rllib/algorithms/algorithm_config.py

```diff
@@ -170,14 +183,15 @@ def __init__(self, algo_class=None):
         self.output_config = {}
         self.output_compress_columns = ["obs", "new_obs"]
         self.output_max_file_size = 64 * 1024 * 1024
+        self.offline_sampling = False
```

Comment:
Moved this here (only used by CRR and CQL, BUT required by RolloutWorker, which had to do a hasattr check :/ )